sys/include/net/gnrc/csma_sender.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2015 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_csma_sender  Helper inerface to send packets via CSMA/CA
+ * @ingroup     net_gnrc
+ * @brief       This interface allows code from layer 2 (MAC) or higher
+ *              to send packets with CSMA/CA, whatever the abilities and/or
+ *              coniguration of a iven radio transceiver device.
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for the CSMA/CA helper
+ *
+ * @author      Kévin Roussel <Kevin.Roussel@inria.fr>
+ */
+
+#ifndef GNRC_CSMA_SENDER_H_
+#define GNRC_CSMA_SENDER_H_
+
+#include "kernel.h"
+#include "net/gnrc/netdev.h"
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * @brief Default Minimal CSMA/CA Backoff Exponent 
+ */
+#define MAC_MIN_BE_DEFAULT              (3U)
+
+/**
+ * @brief Default Maximal CSMA/CA Backoff Exponent 
+ */
+#define MAC_MAX_BE_DEFAULT              (5U)
+
+/**
+ * @brief Default Maximal number of retries for sending
+ *        a given packet with the CSMA/CA method
+ */
+#define MAC_MAX_CSMA_BACKOFFS_DEFAULT   (4U)
+
+/**
+ * @brief CSMA/CA backoff period, in microseconds
+ */
+#define A_UNIT_BACKOFF_PERIOD_uS      (320U)
+
+
+/**
+ * @brief Define a different (non-standard) value for
+ *        the CSMA macMinBE parameter.
+ *
+ * @param[in] val     new value for macMinBE
```

Comment:
@thomaseichinger 
Those names such as `macMinBE` are defined like this in the standard, so I would say it's okay as long as it's just the documentation. What do you think?

IEEE Std 802.15.4-2006 p.146 for example:
> This attribute, which shall only be set by the next higher layer, is dependent upon macMinBE, macMaxBE, macMaxCSMABackoffs and the number of symbols per octet. See 7.4.2 for the formula relating the attributes.